CMP1903_A1_2324/Game.cs

```diff
@@ -17,5 +18,28 @@ internal class Game
 
         //Methods
 
+        public void rollDice(int rollAmount)
+        {
+            int i = 1;
+            int sum = 0;
+            List<int> list = new List<int>();
+
+            while (i <= rollAmount)
```

Comment:
Would a for loop be cleaner and make it clearer that this loop only runs a certain about of times?